### ISSUE TITLE
NREs with null currentMainBody

### ISSUE THREAD

**Issue by Gameslinx** [Gameslinx/Tessellation#12]
Disable Update scripts when the map view is active and the ground can't be seen

**Comment by Poodmund** [comment]
I guess this is the same as:

With Vanilla KSP and Parallax and the Parallax Stock Textures (+ dependencies) I'm seeing a few errors being thrown in the logs. In the KSP.log see the section starting on Line 6473 regarding strings being parsed as floats and also get a bunch of 
```
[LOG 22:36:28.158] [Parallax] Subdivision Error (Adaptive Parallax):
System.NullReferenceException: Object reference not set to an instance of an object
  at PQSModExpansion.PQSMod_Subdivide.OnQuadBuilt (PQ quad) [0x000bd] in <b6b3af8f70dd42d7adac8493f5bd45d3>:0 
```
NullRefs after that section mentioned above before the MainMenu. These NullRefs occur again when loading into flight mode before you actually see the vessel but more importantly, I guess, when you switch to the Planetarium Camera scene NullRef Exceptions are logged per frame starting on Line 20499.
```
[EXC 22:41:27.625] NullReferenceException: Object reference not set to an instance of an object
    ParallaxShader.Shader5.Update () (at <0e962ee0d677445f832ed458f7108ca3>:0)
    UnityEngine.DebugLogHandler:LogException(Exception, Object)
    ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
    UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```
I have attached all the log files of my loading the game, starting a new Sandbox save, going to the Editor (VAB), loading the stock Ariane 5 craft and launching to orbit, going to map mode, then quitting to the main menu.
[Logs.zip](https://github.com/Gameslinx/Tessellation/files/5579445/Logs.zip)

**Comment by Gameslinx** [comment]
Most of the nullrefs occur because the loader is trying to get the height of the camera above the terrain, but the scene is wrong. I've patched this now and I'll update soon.
The errors such as `Input was a string, but it should have been a float: Textures / fogRange` occur because that value is undefined in the config, but that's Parallax skipping over it. I'll be changing the log message to differentiate between "not defined" and "wrong format".
Finally, the AdaptiveParallax errors can be fixed by making that PQSMod only active in the SpaceCenter and Flight scenes

Thanks for the report :)

**Comment by Gameslinx** [comment]
Fixed and will be included in update 1.2.0. Thanks for the report!
